MapboxVision/VisionManager/SessionRecorderProtocol.swift

```diff
@@ -0,0 +1,15 @@
+import CoreMedia
+import Foundation
+
+protocol SessionRecorderProtocol: AnyObject {
+    func stop()
+    func startInternal()
+    func startExternal(withPath path: String)
```

Comment:
After discussion with @chezzdev decided that the original solution is better fits here because it conforms [Swift API design guidlines](https://swift.org/documentation/api-design-guidelines/), so the suggestion is declined